ebpf/c/tracer-ebpf.c

```diff
@@ -379,17 +393,29 @@ static int read_conn_tuple(conn_tuple_t* tuple, tracer_status_t* status, struct
     if (family == CONN_V4) {
         bpf_probe_read(&saddr_l, sizeof(u32), ((char*)skp) + status->offset_saddr);
         bpf_probe_read(&daddr_l, sizeof(u32), ((char*)skp) + status->offset_daddr);
+
+        if (!saddr_l || !daddr_l) {
```

Comment:
The check was previously at the end of the method, I moved it here so that we could exit early and to simplify the process around checking the high/low bits when we have ipv6->ipv4 mapped addresses.